kubeform/cmd/terraform-provider-kubernetes/terraform.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"github.com/hashicorp/terraform/plugin"
+
+	"k8s.io/kubeform/pkg/provider"
```

Comment:
Can I do this in a separate PR? Here is the PR that fixes the import and adds a Makefile - #1416. I would like to keep this PR just for the initial import of the source files without modification so I can easily keep track of things (running diff etc.). The diffs here already look huge.